java/src/jmri/jmrit/logixng/actions/ActionBundle.properties

```diff
@@ -322,11 +322,11 @@ DigitalBooleanMany_Short            = Many
 DigitalBooleanMany_Long             = Many
 
 
-DigitalBooleanOnChange_Short                  = On change
-DigitalBooleanOnChange_Long                   = {0}
-DigitalBooleanOnChange_Trigger_ChangeToTrue   = On change to true
-DigitalBooleanOnChange_Trigger_ChangeToFalse  = On change to false
-DigitalBooleanOnChange_Trigger_Change         = On change
+DigitalBooleanLogixAction_Short                 = Logix Action
+DigitalBooleanLogixAction_Long                  = Logix Action. {0}
+DigitalBooleanLogixAction_Trigger_ChangeToTrue  = On change to true
```

Comment:
Ideally, there would be two sets:  "On change to ..." and "When triggered ...."  The  active set depends on the Logix Execute setting.

Currently, "On change to true" actually means "On change or when triggered true".   This approach would be an easier change.

People migrating from Logix would see the "On change" long description and get confused if the execution mode is "when triggered".

Reply:
Do I understand it correctly that `DigitalBooleanLogixAction_Trigger_Change` should always execute the child action? I think I misunderstood how Logix works in this regard. If so, I could rename `DigitalBooleanLogixAction_Trigger_Change` to `DigitalBooleanLogixAction_AlwaysExecute`.

Reply:
> Do I understand it correctly that `DigitalBooleanLogixAction_Trigger_Change` should always execute the child action? I think I misunderstood how Logix works in this regard. If so, I could rename `DigitalBooleanLogixAction_Trigger_Change` to `DigitalBooleanLogixAction_AlwaysExecute`

Sorry for this reply. I'm a bit tired at the moment. I will look into this.

Reply:
After a second thought, it seems to me that
```
DigitalBooleanLogixAction_Trigger_ChangeToTrue  = On change to true
DigitalBooleanLogixAction_Trigger_ChangeToFalse = On change to false
DigitalBooleanLogixAction_Trigger_Change        = On change
```
should be
```
DigitalBooleanLogixAction_OnTrue  = On True
DigitalBooleanLogixAction_OnFalse = On False
DigitalBooleanLogixAction_Always  = Always
```

Reply:
Logix actions have 3 possibilities:  true (aka then), false (aka else), either.  Whether any action is executed depends on the evaluation state and the "On change/when triggered" conditional setting.

The expanded Logix action text "On change to ..." and "When triggered ..." helps people remember which execution setting was selected.

For Logix to LogixNG import, it works best to keep the phrasing the same.

My latest ideas are:
- True
- False
- Either